packages/terra-dropdown-button/src/SplitButton.module.scss

```
@@ -0,0 +1,78 @@
+@import '~terra-mixins/lib/Mixins';
+
+@import './variables';
+@import './mixins';
+
+:local {
+  .split-button-primary {
+    border-bottom-left-radius: var(--terra-dropdown-button-border-bottom-left-radius, 3px);
```

We'll want to update the naming to be consistent with other variables under this selector.

```suggestion
    border-bottom-left-radius: var(--terra-dropdown-button-split-type-border-bottom-left-radius, 3px);
```